pygeo/constraints/DVCon.py

```diff
@@ -764,6 +764,191 @@ def addThicknessConstraints1D(
             conName, coords, lower, upper, scaled, scale, self.DVGeometries[DVGeoName], addToPyOpt, compNames
         )
 
+    def addProximityConstraints(
+        self,
+        ptList,
+        vecList,
+        surfA,
+        surfB,
+        pointSetKwargsA=None,
+        pointSetKwargsB=None,
+        lower=1.0,
+        upper=3.0,
+        scaled=True,
+        scale=1.0,
+        name=None,
+        addToPyOpt=True,
+        DVGeoName="default",
+        compNames=None,
+    ):
+        r"""
```

Comment:
Any objections to adding a disclaimer to the docstring that says something like:

`
Surfaces must move in the same direction as the constraint orientation to prevent skewness. i.e, if the toothpicks are oriented in the y-direction, the surfaces must only move in the y-direction.
`
</code>

I can see a scenario where two objects both shear and move normal w.r.t each other and these constraints are skewed.  I know this a symptom of the underlying toothpick constraints, but I think it's much easier to achieve this effect with two separate pointsets moving around.